src/test/java/com/example/cat/CatControllerTest.java

```diff
@@ -0,0 +1,25 @@
+package com.example.cat;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@AutoConfigureMockMvc
+@SpringBootTest
+class CatControllerTest {
+
+    @Autowired
+   private MockMvc mockMvc;
+    @Test
+    void GETCAT() throws Exception {
+        mockMvc.perform(MockMvcRequestBuilders.get("/API/miauz"))
+                .andExpect(status().isOk());
+
```

Comment:
Du solltest auch überprüfen das du die Katze im Body der Respone erhälst.
```suggestion
                .andExpect(status().isOk())
               .anddEcpect(content().json("""
               ...
               """))
```